slp_abacus/tests/resources/test_resource_paths.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+path = os.path.dirname(__file__)
+
+# drawio files
```

Comment:
I think this is a copy-paste from the drawio. Here should be the Abacus files used for the tests.